extensions/wikia/Listusers/SpecialListusers_ajax.php

```diff
@@ -4,30 +4,35 @@
  * @package MediaWiki
  * @subpackage SpecialPage
  * @author Piotr Molski <moli@wikia.com>
- * @version: $Id$
  */
-
 class ListusersAjax {
 
 	/**
 	 * ajax response for request params
 	 *
 	 * @author      Piotr Molski <moli@wikia-inc.com>
 	 * @version     1.0.0
-	 * @param       Array   $list
+	 *
+	 * @return AjaxResponse
 	 */
 	public static function axShowUsers ( ) {
-		global $wgRequest, $wgUser, $wgCityId;
-
+		global $wgUser, $wgCityId;
 		wfProfileIn( __METHOD__ );
 
-		$groups 	= $wgRequest->getVal('groups');
-		$username 	= $wgRequest->getVal('username');
-		$edits 		= $wgRequest->getVal('edits');
-		$limit		= $wgRequest->getVal('limit');
-		$offset		= $wgRequest->getVal('offset');
-		$loop		= $wgRequest->getVal('loop');
-		$order		= $wgRequest->getVal('order');
+		$request = RequestContext::getMain()->getRequest();
+
+		$groups 	= $request->getVal('groups');
+		$user_id  	= $request->getInt('userid');
+		$edits 		= $request->getVal('edits');
+		$limit		= $request->getVal('limit');
+		$offset		= $request->getVal('offset');
+		$loop		= $request->getVal('loop');
+		$orders     = explode("|", $request->getVal('order') );
+
+		// FIXME: SUS-3207 - temporarily support searching via user name
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)

[project-id]: wikia:app